src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -244,22 +257,25 @@ private void pausePopup(final int buttonNo) {
 
 	private void playerDies() {
 		State.getTrack().getPlayer().die();
-		sleep(2 * InternalTicker.NANOS_PER_TICK / InternalTicker.E6);
+		sleep(20 * InternalTicker.NANOS_PER_TICK / InternalTicker.E6);
 		letPlayerSurvive();			// Make sure there are no obstacles
 		sleep(LONG);
 	}
 
-	private void deathPopup(final int buttonNo) {
-		if (GameScene.getPopup() == null) {
-			fail("The Death Popup is not available.");
+	private void clickPopup(final int buttonNo) {
+		final AbstractScene scene = ShaftEscape.getScene();
+		if (scene.getPopup() != null) {
```

Comment:
scene.getPopup() should be accessed in a static way.

At least, that's what Eclipse says. And I think the same.

<review,logic